src/pages/home/report/ReportActionCompose.js

```diff
@@ -44,7 +44,8 @@ class ReportActionCompose extends React.Component {
         this.comment = props.comment;
         this.state = {
             isFocused: false,
-            textInputShouldClear: false
+            textInputShouldClear: false,
+            submitButtoncolor: '#ECECEC'
```

Comment:
Styles are not usually stored in state in our codebase. What is more common to see is some condition like a boolean in state and then switching between two different styles defined elsewhere.